.github/workflows/llvm-project-epoch-three.yml

```diff
@@ -0,0 +1,36 @@
+name: stage 3 (epoch 3)
+
+on: workflow_dispatch
+jobs:
+  epoch3:
+    name: stage 3 (epoch 3)
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
```

Comment:
Can this be sunk to right before the push to the registry?

Reply:
I think it can now that the push doesn’t happen in the same step as the build.

Reply:
I opened #20 for this to keep this PR a little more clean.

Reply:
done PTAL